dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -72,7 +72,7 @@
   = form_for(resource, as: resource_name, url: registration_path(resource_name), html: {method: :put, id: 'lti-sync-settings-form'}) do |f|
     = f.hidden_field :lti_roster_sync_enabled
 
-- if @potentially_locked && experiment_value('cpa-partial-lockout', request)
+- if (!@personal_account_linking_enabled || Policies::ChildAccount::ComplianceState.permission_granted?(current_user)) && experiment_value('cpa-partial-lockout', request)
```

Comment:
```suggestion
- if !@personal_account_linking_enabled && experiment_value('cpa-partial-lockout', request)
```
Do we need this `permission_granted?` given that `can_link_new_personal_account?` already calls it?

Reply:
We do actually need this - the distinction is that it's checking for permission being granted even if personal account linking is also enabled. This lets us display the "granted" status even after the account is fully unlocked.